hever/functions.php

```diff
@@ -53,49 +54,67 @@ function hever_setup() {
 			)
 		);
 
+		// Editor color palette.
+		$color_annotations = get_theme_mod( 'colors_manager' )['colors'];
+		$primary          = ( empty( $color_annotations['link'] ) ) ? '#1279BE' : $color_annotations['link'];
+		$secondary        = ( empty( $color_annotations['fg1'] ) ) ? '#FFB302' : $color_annotations['fg1'];
+		$foreground       = ( empty( $color_annotations['txt'] ) ) ? '#303030' : $color_annotations['txt'];
+		$tertiary         = ( empty( $color_annotations['fg2'] ) ) ? '#C5C5C5' : $color_annotations['fg2'];
+		$background       = ( empty( $color_annotations['bg'] ) ) ? '#FFFFFF' : $color_annotations['bg'];
+
+		$foreground_low_contrast  = change_color_luminescence( $foreground, 10 );
+		$foreground_high_contrast = change_color_luminescence( $foreground, -10 );
+		$background_low_contrast  = change_color_luminescence( $background, -10 );
+		$background_high_contrast = change_color_luminescence( $background, 10 );
+
 		// Add child theme editor color pallete to match Sass-map variables in `_config-child-theme-deep.scss`.
 		add_theme_support(
 			'editor-color-palette',
 			array(
 				array(
 					'name'  => __( 'Primary', 'hever' ),
 					'slug'  => 'primary',
-					'color' => '#1279BE',
+					'color' => $primary,
 				),
 				array(
 					'name'  => __( 'Secondary', 'hever' ),
 					'slug'  => 'secondary',
-					'color' => '#FFB302',
+					'color' => $secondary,
 				),
 				array(
-					'name'  => __( 'Dark Gray', 'hever' ),
-					'slug'  => 'foreground-dark',
-					'color' => '#101010',
+					'name'  => __( 'Tertiary', 'hever' ),
+					'slug'  => 'tertiary',
+					'color' => $tertiary,
 				),
 				array(
-					'name'  => __( 'Gray', 'hever' ),
+					'name'  => __( 'Foreground', 'hever' ),
 					'slug'  => 'foreground',
-					'color' => '#303030',
+					'color' => $foreground,
 				),
 				array(
-					'name'  => __( 'Light Gray', 'hever' ),
+					'name'  => __( 'Foreground Low Contrast', 'hever' ),
 					'slug'  => 'foreground-light',
-					'color' => '#757575',
+					'color' => $foreground_low_contrast,
+				),
+				array(
+					'name'  => __( 'Foreground High Contrast', 'hever' ),
+					'slug'  => 'foreground-dark',
+					'color' => $foreground_high_contrast,
 				),
 				array(
-					'name'  => __( 'Lighter Gray', 'varia' ),
-					'slug'  => 'background-dark',
-					'color' => '#C5C5C5',
+					'name'  => __( 'Background', 'hever' ),
+					'slug'  => 'background',
+					'color' => $background,
 				),
 				array(
-					'name'  => __( 'Subtle Gray', 'varia' ),
+					'name'  => __( 'Background High Contrast', 'hever' ),
 					'slug'  => 'background-light',
```

Comment:
Should this be `background-dark` so it's not the same slug as the Background Low Contrast color?

```suggestion
					'slug'  => 'background-dark',
```